### PR TITLE
Bar 29

### PR DESCRIPTION
**Pull request by gabriellenoel**
- Changed Flight Table to look more stylistically similar to Flight List.
- Added Check-In Grouping & Return Flights.

### CODE REVIEW

File: src/components/booking/Booking.jsx [hunk 1]
```diff
@@ -76,13 +93,71 @@ const Booking = () => {
...
+        route: null
+    });
     const [date, setDate] = useState(new Date());
+    const [dateRange, setDateRange] = useState([new Date(), Date.now() + 6.048e+8]);
+    const [startDate, endDate] = dateRange;
     const [origin, setOrigin] = useState("");
     const [dest, setDest] = useState("");
     const [sortBy, setSortBy] = useState("departureTime");
     const [filter, setFilter] = useState("all");
+    const [upgradesPP, setUpgradesPP] = useState(0);
```
Comment (ghost) [inline src/components/booking/Booking.jsx]: Please use descriptive variable names

**Comment by ghost** [comment]
Some tests are failing

File: src/components/booking/Booking.jsx (hunk 1 above)
Comment (gabriellenoel) [reply src/components/booking/Booking.jsx]: Ok, all fixed!

**Comment by ghost** [comment]
I'm not able to display the flight table

**Comment by ghost** [comment]
nvm. got it to display

**Comment by ghost** [comment]
in FlightTable, it looks like you removed the Bootstrap grid breakpoints. We're going to have to add it back for responsiveness at some point.

**Review by ghost — APPROVED** [review]

**Pull request merged**